aipg/sandbox/adapters.py

```
@@ -0,0 +1,108 @@
+            "--cpus",
+            str(self._cpu_quota if self._cpu_quota is not None else 1),
+            "--memory",
+            self._memory_limit,
+            "--pids-limit",
+            str(self._pids_limit),
+            "--security-opt",
+            "no-new-privileges:true",
+            "--user",
+            "65534:65534",  # nobody
+            self._image,
+        ] + shell_cmd
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

(RUF005)